.eslintrc.js

```diff
@@ -184,6 +192,28 @@ module.exports = {
         ],
       },
     },
+    {
+      files: ['src/**', 'pages/**'],
+      rules: {
+        'simple-import-sort/imports': [
+          'warn',
+          {
+            groups: [
+              // External packages come first.
+              ['^react', '^\\w', '^@\\w'],
+              // Cloudscape packages.
+              ['^@cloudscape'],
+              // Things that start with a letter (or digit or underscore), or `~` followed by a letter.
+              ['^~?\\w'],
```

Comment:
Aren't the "Things that start with a letter (or digit or underscore)" already covered by the first group (`^\\w`)? It seems to me that the optional quantifier (`?`) is not needed and this could be just `['^~\\w']` instead.

Reply:
addressed